pkgs/tools/bluetooth/blueman/default.nix

```diff
@@ -31,10 +31,11 @@ in stdenv.mkDerivation rec {
 
   configureFlags = [ (lib.enableFeature withPulseAudio "pulseaudio") ];
 
-  preFixup = ''
+  # This mimics ../../../development/interpreters/python/wrap.sh
+  postFixup = ''
     makeWrapperArgs="--prefix PATH ':' ${binPath}"
```

Comment:
https://github.com/NixOS/nixpkgs/commit/c10af9e744c91dff1ccc07a52a0b57d1e4d339f3

My reasoning was that executables belong in `/bin` and not elsewhere. I did not think of `libexec` at the time.